### PR TITLE
Documentation formatting fix module synchronize

### PR DESCRIPTION
**Pull request by bmillemathias**
##### SUMMARY
* Add markup where needed
* Use FQCN where needed
* Remove reference to a bug in ansible 2.0.0

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
synchronize

### CODE REVIEW

File: plugins/modules/synchronize.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
   dest_port:
     description:
       - Port number for ssh on the destination host.
-      - Prior to Ansible 2.0, the ansible_ssh_port inventory var took precedence over this value.
+      - Prior to Ansible 2.0, the C(ansible_ssh_port) inventory var took precedence over this value.
```
Comment (bmillemathias) [inline plugins/modules/synchronize.py:40]: as this is quite old, could this line be phased out ?

**Comment by saito-hideki** [comment]
recheck

**Comment by saito-hideki** [comment]
Closing and reopening for CI trigger

**Comment by saito-hideki** [comment]
Hi @bmillemathias If it is possible, can you rebase it to the latest main branch?

**Pull request closed without merging**